python/paddle/fluid/distribute_transpiler.py

```diff
@@ -151,10 +152,10 @@ def transpile(self,
             Steps to transpile trainer:
             1. split variable to multiple blocks, aligned by product(dim[1:]) (width).
             2. rename splited grad variables to add trainer_id suffix ".trainer_%d".
-            3. modify trainer program add split_op to each grad variable.
-            4. append send_op to send splited variables to server and fetch
-               params(splited blocks or origin param) from server.
-            5. append concat_op to merge splited blocks to update local weights.
+            3. modify trainer program add split_op and send_op to each grad variable.
```

Comment:
Is our current implementation one `send_op` for each grad variable or one `send_op` for all grad variable?
From my understanding this lines seems to mean one for each, but the implementation seem to be one for all. Maybe we need to have description matching the implementation?

Reply:
Done.